nucypher/characters/control/controllers.py

```diff
@@ -178,27 +71,60 @@ def __init__(self,
         # Internals
         self._transport = None
 
-        # Hard-wire the character's output flow to the Emitter
-        self._internal_controller = character_controller
-        self._internal_controller.emitter = self.emitter
+        self.interface = interface
+
+        def set_method(name):
+
+            def wrapper(request=None, **kwargs):
+                request = request or kwargs
+                return self.handle_request(name, request=request)
+            setattr(self, name, wrapper)
 
-        super().__init__(control_serializer=self._internal_controller.serializer)
+        for method_name in self._get_interfaces().keys():
+            set_method(method_name)
+
+        super().__init__(*args, **kwargs)
 
         self.log = Logger(app_name)
 
+    def _get_interfaces(self):
+        return {
+            name: method for name, method in
+            inspect.getmembers(
+                self.interface,
+                predicate=inspect.ismethod)
+            if hasattr(method, '_schema')
```

Comment:
Perhaps move the suffix to somewhere more conspicuous like a class level variable `Interface.PREDICATE`.